exercises/concept/troll-the-trolls/.docs/instructions.md

```diff
@@ -0,0 +1,90 @@
+# Instructions
+
+Your friend Helma made a small online game that rapidly gained popularity.
+It is called _HellMath_.
+The small community attracted some trolls who make the game and the forums pretty unpleasant.
+Helma has asked you to work on a new permission system to separate "the wheat from the chaff". 
```

Comment:
I'm not sure if phrases like that are easy to understand for somebody who does not speak English fluently. I'd suggest rephrasing the sentence and making it simpler.